samples/javascript/extensions.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
+	<DefaultItemExcludes>**</DefaultItemExcludes>
```

Comment:
Looks like we have a mix of spaces and tabs. Can you convert this to all spaces?

Reply:
Not sure how that happened; will do.